bve/src/parse/function_scripts/mod.rs

```diff
@@ -21,3 +23,41 @@ impl FromKVPValue for ParsedFunctionScript {
         parse_function_script(value).map(|(_, o)| o).ok()
     }
 }
+
+impl PrettyPrintResult for ParsedFunctionScript {
+    fn fmt(&self, indent: usize, out: &mut dyn io::Write) -> io::Result<()> {
+        out.write_all("\n".as_bytes())?;
+        for (idx, instruction) in self.instructions.iter().enumerate() {
+            out.write_all(&vec![b' '; indent * 4])?;
+            out.write_all(format!("{} - ", idx).as_bytes())?;
+            out.write_all(
+                match instruction {
```

Comment:
Does Rust really not have a good way to print an enum?

Reply:
The other way is to break out Serde_Plain, which I thought was overkill here.